### PR TITLE
Add .gitignore file

### PR DESCRIPTION
**Pull request by hupreti**
Adding .gitignore file, necessary for future development purposes

### CODE REVIEW

**Review by vbaddi — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Distribution / packaging
```
Comment (anujgupt-github) [inline .gitignore:6]: why do we need this file here?

Comment (hupreti) [reply .gitignore:6]: we must have .gitignore file in repo, kind of standard template keeping in mind the future development of the project

**Comment by quic-shagun** [comment]
Any particular reason this is not merged yet?

**Review by anujgupt-github — APPROVED** [review]

**Pull request merged**